doozerlib/cli/detect_embargo.py

```diff
@@ -45,6 +45,29 @@ def detect_in_nvr(runtime: Runtime, nvrs, as_yaml, as_json):
     embargoed_builds = detect_embargoes_in_nvrs(runtime, nvrs)
     print_result_and_exit(embargoed_builds, None, None, as_yaml, as_json)
 
+@detect_embargo.command("images", short_help="Detect embargoed fixes in given images")
```

Comment:
The verb name `detect-embargo images` sounds like it is to detect embargoes in a list of images, but what it actually does is finding images that changed to embargoes from non-embargoes.